### ISSUE TITLE
Mat/Par - Parental leave approval with a future birth date

### ISSUE THREAD

**Issue by fazil-ey** [bcgov/digital-journeys#1663]
Currently if the non-birthing parent is taking leave and the baby isn't born yet, the supervisor cannot sign off the form until the date of birth is confirmed.  Payroll is concerned that if they wait for the form with the confirmed date of birth, it may not get processed in time therefore causing an overpayment. Currently the employee would put the estimated date of birth and the dates they want to take so at least something is entered into PeopleSoft and can be amended if needed.  They are concerned if they submit it at the last minute, it may not be processed in time.  

**_AC_**
- [ ] Remove the validation on the supervisor's end that they can approve only after the child's birth date is in the past
- [ ] If taking only parental leave change the content on Parental Leave - Start Date 

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOCt6QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9b05a845961009ed049d8adad6a466e3990f20f2/image.png" alt="image.png" />

---------------

**Potential Solutions**
1. MVP - remove the validation that supervisor can approve only after child's date of birth has passed. employees can submit and supervisors can approve, and employees have to remember to get it corrected if they start on a different date (earlier or later) otherwise they will be overpaid or underpaid 

2. Nice to have - Ask in the form if the employee has already had the baby or not. If not, then clearly ask only for estimated birth date and estimated leave start date. Add clear instructions that they MUST submit the birth certificate after the child is born and correct their leave start date if needed otherwise they will not be paid correctly. We can also send a reminder email for those who put in estimated dates. (#1664)

**Comment by ayushdamani** [comment]
Confirmed updated in TEST - thanks Andrew!
Moving to PO

**Comment by fazil-ey** [comment]
validation removed, works well
